CHANGELOG.md

```diff
@@ -30,7 +30,9 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
     - Secured clusters configured to use KernelModule collection will automatically switch to EBPF
 
 ### Deprecated Features
-Vulnerability Management 1.0 sections Image CVEs, Image Components, Images, Deployments, and Namespaces are deprecated and will be removed in the future. Once removed, use Vulnerability Management 2.0 for managing workload vulnerabilities.
+- Vulnerability Management 1.0 sections Image CVEs, Image Components, Images, Deployments, and Namespaces are deprecated and will be removed in the future. Once removed, use Vulnerability Management 2.0 for managing workload vulnerabilities.
+- Custom Security Context Constraints (SCC) (e.g.: `stackrox-collector`, `stackrox-admission-control`, `stackrox-sensor`) are deprecated and will be removed in the future.
```

Comment:
Could we add a warning in Helm that these should be disabled?
Does something prevent us from disabling SCCs directly by default for customers who not opted-in explicitly?

Reply:
> Could we add a warning in Helm that these should be disabled?

Great idea! Will add this

Reply:
> Does something prevent us from disabling SCCs directly by default for customers who not opted-in explicitly?

Yes, it may happen that those SCCs are being used by other apps running on the cluster (non-ACS). If we delete them, we may break these apps. The chance is low, but it still exists.

Reply:
@SimonBaeumer I won't make it today with the Helm warning. Maybe there is enough time tomorrow before the freeze? Any help is welcomed here.

Reply:
Merging sounds fine +1